src/main/java/phenex/storage/Storage.java

```diff
@@ -0,0 +1,62 @@
+package phenex.storage;
+
+import phenex.task.Task;
+import phenex.task.TaskList;
+import phenex.ui.Ui;
+import phenex.util.Parser;
+
+import java.io.File;
+import java.io.FileWriter;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.io.IOException;
+
+/**
+ * Storage class which encapsulates the storage for Phenex.
+ */
+public class Storage {
+    /** Encapsulates the filePath in storage. */
+    protected final Path filePath;
+    /** Encapsulates the File in storage. */
+    protected File file;
+
+    public Storage(Path filePath) {
+        this.filePath = filePath;
+
+        if (!Files.exists(this.filePath)) {
```

Comment:
I like how you’ve handled the initialization of the storage file. It’s a good practice to ensure that necessary resources are in place before proceeding.